[CLI]: specify framework version for project creation

baquan1708 opened the pull request:
## Summary
This pull request enhances the CLI tool mbc by adding the capability to specify the framework version for project creation. The update provides flexibility for users to choose between using the latest framework version, specifying a version, or matching a version prefix.

## Changed
Users can now create projects with the following commands:
- `mbc new`
  - Creates a new project using a default name with the latest framework version.
- `mbc new projectName`
  - Creates a new project with the specified projectName using the latest framework version.
- `mbc new projectName@version`
  - If the specified version exists, the CLI uses that exact version.
  - If the provided version is a prefix, the CLI uses the latest version matching that prefix.
  - If no matching version is found, the CLI logs an error and provides a list of available versions for the user.

koichimurakami commented:
@baquan1708 
Thank you.
Could you write readme.md about it?
and could you create unit test?

baquan1708 commented:
Yes, I will update readme.md and add unit test for this feature.

baquan1708 commented:
@koichimurakami I've just added the unit tests and updated the README in recent commits. Could you review it?

Note: If this pull request is merged, should I also update the CLI section in the [documentation repository](https://github.com/mbc-net/mbc-cqrs-serverless-doc)?
